modules/identity/src/Volo.Abp.Identity.Application/Volo/Abp/Identity/AbpIdentityApplicationModuleAutoMapperProfile.cs

```diff
@@ -11,9 +12,11 @@ public AbpIdentityApplicationModuleAutoMapperProfile()
 
             CreateMap<IdentityRole, IdentityRoleDto>()
                 .MapExtraProperties();
-            
+
             CreateMap<IdentityUser, ProfileDto>()
+                .Ignore(x=>x.IsExternalLoggedIn)
+                .Ignore(x=>x.HasPassword)
```

Comment:
Instead of ignoring `HasPassword`, you can map it with a custom lambda expression that checks `PasswordHash != null`. In this way, you don't need to manual mapping later.